lib/services/servermanagement/generate.cmd

```diff
@@ -0,0 +1,17 @@
+::

```

Comment:
Ah dammit. Yeah, I knew that.

I had that in my other branch, and forgot to pull it over. 

FIXED.